Makefile

```diff
@@ -1,28 +1,113 @@
 PYTHON=`which python`
+PYTHON2=`which python2`
+PYTHON3=`which python3`
+PY2DSC=`which py2dsc`
+
+topdir := $(realpath $(dir $(lastword $(MAKEFILE_LIST))))
+topbuilddir := $(realpath .)
+
 DESTDIR=/
-PROJECT=python-nodefacade
+PROJECT=$(shell python $(topdir)/setup.py --name)
```

Comment:
You're not going to like this, but because of the way py2dsc works (well, I assume it's py2dsc doing it...never worked it out!) you end up with a package called python-python-nodefacade (see https://jenkins.rd.bbc.co.uk/blue/organizations/jenkins/apmm-repos%2Fnmos-node/detail/PR-12/1/artifacts). I think either the name in setup.py needs to become "nodefacade" (and accept the potential breakage that ensues) or some more Makefile hacking is needed.